modules/gapi/test/common/gapi_operators_tests_inl.hpp

```diff
@@ -77,6 +77,103 @@ TEST_P(NotOperatorTest, OperatorAccuracyTest)
         EXPECT_EQ(0, cvtest::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
     }
 }
+
+namespace for_test
+{
+class Foo {};
+
+int operator&(Foo, int);
+int operator&(Foo, int) { return 1; }
```

Comment:
IMHO no need in these forward declarations. As well `inline`  specifier is required.

Reply:
Yes, you're right, thanks!
reworked